kura/distrib/src/main/resources/docker-x86_64-nn/deb/control/control

```diff
@@ -5,7 +5,8 @@ Priority: low
 Depends: dos2unix, unzip, 
  telnet, bluez-hcidump, 
  bluez, chrony, ntpdate, 
- openjdk-8-jre-headless, cron
+ openjdk-8-jre-headless | temurin-8-jdk | openjdk-17-jre-headless | temurin-17-jdk,
```

Comment:
Could these alternative be too restrictive? I've seen other packages use the following:

```
 openjdk-8-jdk | openjdk-8-jdk-headless | openjdk-8-jre | openjdk-8-jre-headless 
```

instead of only

```
openjdk-8-jre-headless
```

to cover all the bases.

Reply:
Done some tests and `openjdk-8-jre-headles` is a subset of `openjdk-8-jre`; installing `openjdk-8-jre` the headless dependency is satisfied.

Reply:
Can we confidently say that the same goes for `openjdk-8-jdk`? As in: if we specify in our `Depends` only `openjdk-8-jre-headless`, `openjdk-8-jdk` satisfies that dependency?

Reply:
Yes

Reply:
After some testing we determined that this is true.